nebu/models/utils.py

```diff
@@ -50,8 +50,10 @@ def convert_to_model_compat_metadata(metadata):
     md.setdefault('cnx-archive-uri', '{}@{}'.format(md['id'], md['version']))
     md.pop('id')
     # FIXME cnx-epub has an issue rendering and parsing license_text set to
-    #       None, so hard code it to 'CC BY' for now.
-    md.setdefault('license_text', 'CC BY')
+    #       None, so hard code it to
+    #       'Creative Commons Attribution License (ASSUMED)' for now.
+    md.setdefault('license_text',
```

Comment:
`setdefault` will not overwrite values that are `None`. I think this should be
```python
if md.get('license_text', None) is None:
    md['license_text'] = 'Creative Commons Attribution License (ASSUMED)'
```

The problem is that now `license_text` should always arrive here with a value of None or a string. That setdefault was meant to prevent unset values from bubbling up, but it will not replace `None` values.